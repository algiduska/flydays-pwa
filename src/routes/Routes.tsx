import React from 'react';
import { Route, Switch } from 'react-router-dom';
import HomePage from '../pages/HomePage/HomePage';
import AboutPage from '../pages/AboutPage/AboutPage';

const Routes = () => (
  <Switch>
    <Route exact path="/" component={HomePage} />
    <Route exact path="/about" component={AboutPage} />
  </Switch>
);

export default Routes;
