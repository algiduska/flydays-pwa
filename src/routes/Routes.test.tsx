import React from 'react';
import { shallow } from 'enzyme';
import Routes from './Routes';

describe('Routes', () => {
  const wrapper = shallow(<Routes />);

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
