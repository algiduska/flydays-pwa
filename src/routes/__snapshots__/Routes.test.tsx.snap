// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should match the snapshot 1`] = `
<Switch>
  <Route
    exact={true}
    path="/"
    render={[Function]}
  />
  <Route
    component={[Function]}
    exact={true}
    path="/about"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/dashboard"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/events"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/sign-up"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/error/:errorCode"
  />
  <Route
    render={[Function]}
  />
</Switch>
`;
